### ISSUE TITLE
Create board@recoverydharma.org

### ISSUE THREAD

**Issue by KrisRecoveryDharma** [RecoveryDharma/Infrastructure#6]
## 🤔 Tell Us About the Feature

Create a new email address:  board@recoverydharma.org
Connect it to info@recoverydharma.org (is that possible?)

Thanks!

**Comment by catpanclub** [comment]
Can you clarify what you mean by "connect?"

"info@" is already an alias for "recoverdharmainfo@" so the only way to do so would be to have all mail that comes into "board@" also forwarded to "info@". Actions taken in one account are not always reflected in the other.

Here's what I mean by that, and what I see as the downfall of relying on forwarding rather than just having someone dedicated to checking an email account (one person, ideally). If email to "info@dirk.com" is forwarded to "dirk@dirk.com," and I just use that second account, I can't reply easily to emails (because I would be replying not to the original sender, but info@dirk.com). Also, if another person takes over info@dirk.com someday, and I always used the other email account to (awkwardly) reply to and manage inquiries, no history of any of that will be stored in the original account.

I can do a "too long, didn't read" version of this if you want. And I can create the email address, but I'll need an email address and someone to activate and manage the password.

**Comment by KrisRecoveryDharma** [comment]
Hi Dirk,

Ah yes, that makes sense. Thank you for clarifying. Lacey will be checking
it, so her email would be the one.

We created it because we were experiencing a repeat pattern that people did
not believe that info@ was reaching the board.  Explanations did not seem
to work. So, we listened and decided to go ahead and create board@  I asked
the board if they wanted to stop using info@, but they want to keep it.

Thanks,

Kris
